content/desktop/installing-and-configuring-github-desktop/configuring-and-customizing-github-desktop/configuring-git-for-github-desktop.md

```diff
@@ -1,62 +1,114 @@
+
+{% data variables.product.prodname_desktop %} also allows you to change the default branch name that you would like to use when creating new repositories. By default, {% data variables.product.prodname_desktop %} uses `main` as the default branch name in any new repositories you create.
 
 {% tip %}
 
 **Tip**: Anyone will be able to see the email address in your Git configuration if you make public commits. For more information, see "[Setting your commit email address](/articles/setting-your-commit-email-address/)."
 
 {% endtip %}
 
+## Configuring your commit details
+
 {% mac %}
 
-{% data reusables.desktop.sign-in-choose-product %}
-{% data reusables.user-settings.access_settings %}
-{% data reusables.user-settings.emails %}
-{% data reusables.desktop.copy-email-git-config %}
-{% data reusables.desktop.return-to-desktop %}
 {% data reusables.desktop.mac-select-desktop-menu %}
 7. In the Preferences window, click **Git**.
   ![The Git Pane in the Preferences menu](/assets/images/help/desktop/mac-select-git-pane.png)
 {% data reusables.desktop.name-field-git-config %}
   ![The name field of the Git configuration](/assets/images/help/desktop/mac-name-git-config.png)
-{% data reusables.desktop.paste-email-git-config %}
-  ![Pasted email address in Git configuration field](/assets/images/help/desktop/mac-email-git-config.png)
+{% data reusables.desktop.select-email-git-config %}
+  ![Select email address in Git configuration field](/assets/images/help/desktop/mac-email-git-config.png)
 {% data reusables.desktop.click-save-git-config %}
   ![Save button in Git configuration field](/assets/images/help/desktop/mac-save-git-config.png)
 
 {% endmac %}
 
 {% windows %}
 
-{% data reusables.desktop.sign-in-choose-product %}
-{% data reusables.user-settings.access_settings %}
-{% data reusables.user-settings.emails %}
-{% data reusables.desktop.copy-email-git-config %}
-{% data reusables.desktop.return-to-desktop %}
 {% data reusables.desktop.windows-choose-options %}
 8. In the Options window, click **Git**.
 ![The Git Pane in the Options menu](/assets/images/help/desktop/windows-select-git-pane.png)
 {% data reusables.desktop.name-field-git-config %}
   ![The name field of the Git configuration](/assets/images/help/desktop/windows-name-git-config.png)
-{% data reusables.desktop.paste-email-git-config %}
-  ![Pasted email address in Git configuration field](/assets/images/help/desktop/windows-email-git-config.png)
+{% data reusables.desktop.select-email-git-config %}
+  ![Select email address in Git configuration field](/assets/images/help/desktop/windows-email-git-config.png)
 {% data reusables.desktop.click-save-git-config %}
   ![Save button in Git configuration field](/assets/images/help/desktop/windows-save-git-config.png)
 
 {% endwindows %}
 
+## Overriding the email address GitHub Desktop uses for a repository
```

Comment:
You can also change your name here too, so let's be a bit more general and also more directly contrast with the name for the first section, to help folks differentiate between them.
```suggestion
## Configuring different author information for an individual repository
```